cmd/podman/play_kube.go

```diff
@@ -117,12 +118,25 @@ func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
 	podOptions = append(podOptions, libpod.WithInfraContainerPorts(podPorts))
 
 	// Create the Pod
-	pod, err := runtime.NewPod(ctx, podOptions...)
+	pod, err = runtime.NewPod(ctx, podOptions...)
 	if err != nil {
 		return err
 	}
-	// Print the Pod's ID
-	fmt.Println(pod.ID())
+	// We've now created a resource that will need to be cleaned up if we fail.
+	defer func() {
+		if err != nil {
+			if containers != nil {
+				for _, ctr := range containers {
```

Comment:
Actually, do we need to do this if we're already force-removing the pod? That ought to evict all remaining containers in it anyways.

Reply:
Ope yeah you're right

Reply:
Fixed